miner/miner.go

```diff
@@ -425,8 +425,25 @@ func (m *Miner) mineOne(ctx context.Context, base *MiningBase) (*types.BlockMsg,
 		return nil, xerrors.Errorf("failed to get mining base info: %w", err)
 	}
 	if mbi == nil {
+		log.Warnf("mineOne: unexpectedly nil MiningBaseInfo for round %d, off tipset %d/%s", round, base.TipSet.Height(), base.TipSet.Key().String())
 		return nil, nil
 	}
+
+	var winner *types.ElectionProof
+
+	// always write out a log from this point out
+	defer func() {
+		log.Infof(
```

Comment:
I would say use `Infow` so the infos are structured logs

Reply:
@Kubuxu done. I also added the VRF debug directly into the logs, it isn't very noisy at all, see below